public/content/developers/docs/transactions/index.md

```diff
@@ -160,7 +160,9 @@ The base fee will be burned **-0.00399 ETH**
 
 Validator keeps the tip **+0.000210 ETH**
 
-Gas is required for any smart contract interaction too.
+In case of an interaction with a smart contract there is 2 cases possible:
+1. A `view` or `pure` function (as described in [solidity documentation](https://docs.soliditylang.org/en/v0.8.23/contracts.html#view-functions)) is called from an EOA. In that specific case, no gas will be paid by the caller and the result will be returned. The underlying RPC call for this scenario is [eth_call](https://ethereum.org/en/developers/docs/apis/json-rpc#eth_call)
```

Comment:
I see where you're going with this but it doesn't seem appropriate for where it is on the page. I might break this into it's own h3 header, ie `### Smart contract interactions {#smart-contract-interactions}`

I would probably leave something similar to the comment you deleted as well though, just updated. Maybe "Gas is require for any transaction that alters the state of a smart contract" instead... weaving in the rest of the content. 

Also, the link to `eth_call` should be relative: `/developers/docs/apis/json-rpc#eth_call`, and consider wrapping the term itself in single-backticks to style it as code.

Reply:
Hey @wackerow, thanks for your feedback, I made changes accordingly.

I agree for the h3 header, but in that case I would suggest to rename the `### On gas {#on-gas}` to `### Regular transactions {#regular-transactions}`. Let me know what's your opinion is on this